jwcrypto/tests.py

```diff
@@ -19,14 +19,17 @@
 from jwcrypto.common import base64url_decode, base64url_encode
 from jwcrypto.common import json_decode, json_encode
 
-# RFC 7517 - A.1
+# RFC 7517, RFC 8037 - A.1
```

Comment:
Please do not mix RFC tests, add a new variable specific for RFC8037 keys unless that RFC reuses keys froma  previous RFC.

Reply:
Sorry, I'll do that.